test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -50,12 +49,7 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 
 		It("should generate and run a valid OLM bundle and packagemanifests", func() {
 			By("building the bundle")
-			err := tc.Make("bundle")
-			Expect(err).NotTo(HaveOccurred())
-
-			By("validating the bundle")
-			bundleValidateCmd := exec.Command(tc.BinaryName, "bundle", "validate", "bundle")
-			_, err = tc.Run(bundleValidateCmd)
```

Comment:
Why is this removed?

Reply:
`bundle validate` is already called by `make bundle`.